osu.Game/Screens/Edit/Compose/Components/Timeline/TimelineArea.cs

```diff
@@ -154,6 +156,6 @@ private void load(OverlayColourProvider colourProvider)
             Timeline.TicksVisible.BindTo(ticksCheckbox.Current);
         }
 
-        private void changeZoom(float change) => Timeline.Zoom += change;
+        private void changeZoom(float change) => Timeline.Zoom += change * Timeline.MaxZoom * zoom_button_sensitivity;
```

Comment:
I'd rather this be moved to a method inside `ZoomableScrollContainer` so the `const` / `MaxZoom` interaction doesn't need to be repeated here.

Reply:
I also don't think the math is very correct on this. If the intent was that the "step" of the zoom should be 0.02x (or 2%) of the allowable range, then this isn't accounting for `MinZoom` correctly. Should likely be something along the lines of `MinZoom + (MaxZoom - MinZoom) * zoom_button_sensitivity`.